crates/sui/src/client_commands.rs

```diff
@@ -753,8 +753,8 @@ impl SuiClientCommands {
                 let rpc = context.config.get_active_env()?.rpc.clone();
                 let _command_result =
                     sui_replay::execute_replay_command(Some(rpc), false, false, None, cmd).await?;
-
-                SuiClientCommandResult::ProfileTransaction
+                // this will be displayed via trace info, so no output is needed here
```

Comment:
Not all the output is displayed via trace info, does the returned output (tx effects) get displayed still?

Reply:
This change shouldn't affect how the tx effects get displayed -- I'm assuming that's happening within `execute_replay_command` -- because it's replacing a number of `SuiClientCommandResult` cases that all return no output (see L1904-L1907 below) with one generic case.

This does raise a question whether this command should produce no output actually, or whether we should be using the `CommandResult` enum for `ProfileTransaction` to return the effects and then display them from here.